packages/server/src/question/question.controller.ts

```diff
@@ -150,4 +154,34 @@ export class QuestionController {
       );
     }
   }
+
+  @Post(':questionId/notify')
+  async notify(
+    @Param('questionId') questionId: number,
+    @UserId() userId: number,
+  ): Promise<void> {
+    const question = await QuestionModel.findOne(questionId, {
+      relations: ['queue'],
+    });
+
```

Comment:
can we add a TODO here to check that the user is a TA of the question's queue or currently helping that question?